### PR TITLE
Numpy-exercise-1

### PR DESCRIPTION
**Pull request by sumit1136**

### CODE REVIEW

**Comment by sumit1136** [comment]
I have downloaded ipynb file and uploaded in previous pull request. I have done using proper method

File: submission/SUMIT-2020-09-26.ipynb [hunk 1]
```diff
@@ -0,0 +1,979 @@
...
+      "metadata": {
+        "id": "C3wRTspr9vh_",
+        "outputId": "51e7ceac-2568-428f-9461-210ed87da156",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 34
+        }
+      },
+      "source": [
+        "arr01d=np.zeros((1,10))\n",
```
Comment (heyparitosh) [inline submission/SUMIT-2020-09-26.ipynb]: You have created 2d array. Correct it.

File: submission/SUMIT-2020-09-26.ipynb [hunk 2]
```diff
@@ -0,0 +1,979 @@
...
+        "scrolled": true,
+        "id": "efl7_9aK9viE",
+        "outputId": "f1424e77-20a2-438d-a76c-5f9d751e9891",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 34
+        }
+      },
+      "source": [
+        "arr11d=np.ones((1,10))\n",
```
Comment (heyparitosh) [inline submission/SUMIT-2020-09-26.ipynb]: You have created 2d array. Correct it.

File: submission/SUMIT-2020-09-26.ipynb [hunk 3]
```diff
@@ -0,0 +1,979 @@
...
+      "metadata": {
+        "id": "wPDfBofF9viL",
+        "outputId": "564735dd-0955-4282-a065-662c21e6be09",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 34
+        }
+      },
+      "source": [
+        "arr=5*np.ones((1,10),dtype=np.int)\n",
```
Comment (heyparitosh) [inline submission/SUMIT-2020-09-26.ipynb]: You have created 2d array. Correct it.

File: submission/SUMIT-2020-09-26.ipynb [hunk 4]
```diff
@@ -0,0 +1,979 @@
...
+      "metadata": {
+        "id": "cRkc58PE9vi3",
+        "outputId": "bb6f9dbf-f5c4-481f-84f7-1601eb5b2805",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 68
+        }
+      },
+      "source": [
+        "arr_num=np.random.normal(0,1,15)\n",
```
Comment (heyparitosh) [inline submission/SUMIT-2020-09-26.ipynb]: Create an array of 25 random numbers. You have done for 15.

File: submission/SUMIT-2020-09-26.ipynb [hunk 5]
```diff
@@ -0,0 +1,979 @@
...
+        "----"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "3XdIkLBx9vg8"
+      },
+      "source": [
+        "# <font color=\"Green\">NumPy<font/>"
```
Comment (heyparitosh) [inline submission/SUMIT-2020-09-26.ipynb]: Could you please remove <font> tag as in latest changes to the notebook, it is being deleted.

File: submission/SUMIT-2020-09-26.ipynb [hunk 6]
```diff
@@ -0,0 +1,979 @@
...
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "55No34zu9vg4"
+      },
+      "source": [
+        "# <font color=\"Red\"> GRAY INTERFACE </font>\n",
+        "\n",
+        "## <font color=\"Blue\"> HackSlash </font>"
```
Comment (heyparitosh) [inline submission/SUMIT-2020-09-26.ipynb]: Could you please remove <font> tag as in latest changes to the notebook, it is being deleted.

File: submission/SUMIT-2020-09-26.ipynb [hunk 7]
```diff
@@ -0,0 +1,979 @@
...
+        "<a href=\"https://colab.research.google.com/github/sumit1136/train-ml/blob/master/submission/SUMIT-2020-09-26.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "55No34zu9vg4"
+      },
+      "source": [
+        "# <font color=\"Red\"> GRAY INTERFACE </font>\n",
```
Comment (heyparitosh) [inline submission/SUMIT-2020-09-26.ipynb]: Could you please remove <font> tag as in latest changes to the notebook, it is being deleted.

**Review by heyparitosh — CHANGES_REQUESTED** [review]
Correct few things, rest is all fine!

File: submission/SUMIT-2020-09-26.ipynb (hunk 5 above)
Comment (sumit1136) [reply submission/SUMIT-2020-09-26.ipynb]: Can please explain tag. I don't know about it. And how to remove it.

File: submission/SUMIT-2020-09-26.ipynb (hunk 7 above)
Comment (sumit1136) [reply submission/SUMIT-2020-09-26.ipynb]: Please explain tag. I don't know about it. I have done few changes.

Comment (heyparitosh) [reply submission/SUMIT-2020-09-26.ipynb]: Ok. I will edit that and merge.

**Review by heyparitosh — APPROVED** [review]
Good job! 😄

**Pull request merged**